group.py

```diff
@@ -2,4 +2,47 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {
+    'person': {
+        'age': '',
+        'job': '',
+        'connections': {
+            'connections_name': 'connection_type'
```

Comment:
A list for the connection type would allow multiple connections

Reply:
Ah yeah, did think about this after the pull request